src/unix/bsd/apple/mod.rs

```diff
@@ -250,6 +250,12 @@ s! {
         pub l_type: ::c_short,
         pub l_whence: ::c_short,
     }
+    pub struct sf_hdtr {
```

Comment:
Typically each struct has a blank line between it and the previous one (here and below)